.claude/skills/youtube-collector/scripts/fetch_transcript.py

```
@@ -0,0 +1,138 @@
+                try:
+                    transcript = transcript_list.find_generated_transcript(['en'])
+                except NoTranscriptFound:
+                    pass
+
+        # 4. 아무 자막이나 가져오기
+        if not transcript:
+            try:
+                for t in transcript_list:
+                    transcript = t
+                    break
+            except:
```

Ruff (0.14.11)

Do not use bare `except`

(E722)